File: beecrowd/1101.py
maior = []
menor = []
cont = 0
while True:
    duplas = input().split()
    n1 = int(duplas[0])
    n2 = int(duplas[1])
    if n1 <= 0 or n2 <= 0:
        break
    cont += 1
    if n1 > n2:
        maior.append(n1)
        menor.append(n2)
    else:
        maior.append(n2)
        menor.append(n1)

for k in range(0, cont):
    soma = 0
    for v in range(menor[k], maior[k] + 1):
        print(f'{v} ', end='')
        soma += v
    print(f'Sum={soma}')

File: beecrowd/1101.c
#include <stdio.h>

int main() {

    int m, n, maior, menor, soma;

    m = n = 5;
    
    while(m > 0 && n > 0) {
        soma = 0;
        scanf("%d%d", &m, &n);
        if(m > 0 && n > 0) {
        maior = m;
        menor = n;
        if(m < n) {
            maior = n;
            menor = m;
        }
        while(menor <= maior) {
            printf("%d ", menor);
            soma += menor;
            menor += 1;
        }
        printf("Sum=%d\n", soma);
        }
    }

    return 0;
}